ember-app/app/controllers/issue.js

```diff
@@ -1,7 +1,15 @@
 import Ember from 'ember';
+import IssueEvent from "app/mixins/events/issue";
+import Messaging from "app/mixins/messaging";
 
-var IssueController = Ember.Controller.extend({
+var IssueController = Ember.Controller.extend(
+  IssueEvent, Messaging, {
   needs: ["application"],
+
+  //Subscribes to Messages in the Route after the model is set
+  subscribeDisabled: true,
```

Comment:
Yah, that sounds pretty reasonable - I had thought that the controller would call ```willDestroy``` in between view changes, ill make the changes